pkgs/desktops/xfce/applications/xfce4-panel-profiles/default.nix

```diff
@@ -0,0 +1,29 @@
+{ mkXfceDerivation, lib, python3, intltool, gettext,
+ gtk3, libxfce4ui, libxfce4util, pango, harfbuzz, gdk-pixbuf, atk }:
+
+let
+  pythonEnv = python3.withPackages(ps: [ ps.pygobject3 ]);
+  makeTypelibPath = lib.makeSearchPathOutput "lib/girepository-1.0" "lib/girepository-1.0";
+in mkXfceDerivation {
+  category = "apps";
+  pname = "xfce4-panel-profiles";
+  version = "1.0.13";
+
+  sha256 = "sha256-B3Q5d3KBN5m8wY82CIbIugJC8nNS+OcgKchn+TGrDhc=";
+
+  nativeBuildInputs = [ intltool gettext ];
+  propagatedBuildInputs = [ pythonEnv ];
+
+  configurePhase = ''
+    ./configure --prefix=$out
+  '';
```

Comment:
Use configureFlags

Reply:
The script doesn't support all flags. Probably because it's a python script.

With `configureFlags = [ "--prefix=${placeholder "out"}" ];` build fails with:
```configure flags: --prefix=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13 --bindir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/bin --sbindir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/sbin --includedir=/nix/store/c4bc8nsfib6pf081zzmj6zchgc9giqc6-xfce4-panel-profiles-1.0.13-dev/include --oldincludedir=/nix/store/c4bc8nsfib6pf081zzmj6zchgc9giqc6-xfce4-panel-profiles-1.0.13-dev/include --mandir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/share/man --infodir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/share/info --docdir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/share/doc/xfce4-panel-profiles --libdir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/lib --libexecdir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/libexec --localedir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/share/locale --enable-maintainer-mode --prefix=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13
Unknown option --bindir=/nix/store/lnlhrk2kwsx5d96cxj1aqdmhh2vgdnsr-xfce4-panel-profiles-1.0.13/bin
Supported options are:
    --help                          print this help and exit
    --prefix=<path>                 specify installation prefix
        <path>/bin - will hold all executables
        default <path> is /usr/local
    --python=<python executable>    specify python version
        python or python3
        default <python executable> is python3
    --docdir=<path>    specify docdir
        default <path> is $PREFIX/share/doc/xfce4-panel-profiles
builder for '/nix/store/g8jkak2i29nmjmhhvk25nc96ds03sffn-xfce4-panel-profiles-1.0.13.drv' failed with exit code 1
```